sqldelight-gradle-plugin/src/test/fixtures/custom-class-works-fine/expected/com/test/UserModel.java

```diff
@@ -47,6 +47,11 @@ public UserMarshal(ColumnAdapter<User.Money> balanceAdapter) {
       this.balanceAdapter = balanceAdapter;
     }
 
+    public UserMarshal(UserModel copy, ColumnAdapter<User.Money> balanceAdapter) {
```

Comment:
its impossible to avoid since you have to provide these adapters no matter what. I like this way more since it forces you to put the entity in first preventing situations like 

```
db.insert(User.TABLE_NAME, User.marshal()
  ._id(2)
  .copy(user)
  .asContentValues());
```